spec/requests/api/v0.1/inventory_spec.rb

```diff
@@ -25,9 +25,397 @@
 
         post("/topological_inventory/ingress_api/0.0.2/inventory", :params => body, :headers => headers)
 
-        expect(log_line).to     include("[FILTERED]")
+        expect(log_line).to include("[FILTERED]")
         expect(log_line).to_not include("HUMONGOUS_PAYLOAD")
       end
     end
+
+    context "validate post" do
+      before :each do
```

Comment:
Minor, but :each is the default, and so is not necessary

Reply:
:+1: